blocklytest/src/androidTest/java/com/google/blockly/android/ui/AbstractBlockViewTest.java

```diff
@@ -77,6 +77,16 @@ protected boolean hitTest(MotionEvent event) {
             }
 
             @Override
+            protected boolean isInHorizontalRangeOfBlock(int x) {
+                return false;
+            }
+
+            @Override
+            protected boolean coordinatesAreOnBlock(int x, int y) {
+                return false;
```

Comment:
this is fine for now, but I suspect unit tests for hit testing will require the ability to mock this method.